crates/host/src/module.rs

```diff
@@ -4,7 +4,6 @@ use bimap::BiMap;
 use bytes::BufMut;
 use bytes::Bytes;
 use bytes::BytesMut;
-use holochain_wasmer_common::WasmError;
```

Comment:
No longer needed, showing that the fix is working. We'll get a bunch of warnings like this in Holochain and hApps when we release this